aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -904,9 +916,12 @@ bool try_configs(
       benchmark_cache.update(key, plan);
       return true;
     } catch (cudnn_frontend::cudnnException& e) {
+      TORCH_WARN("Plan failed with a cudnnException: ", e.what());
     } catch (CuDNNError& e) {
+      TORCH_WARN("Plan failed with a CuDNNError: ", e.what());
     } catch (c10::OutOfMemoryError& e) {
       (void)cudaGetLastError(); // clear CUDA error
+      TORCH_WARN("Plan failed with an OutOfMemoryError: ", e.what());
```

Comment:
nit: Since this covers 3/4 paths already, how about  adding the same warnings to the 4th (`try_configs_fused`?)

Reply:
Totally, modified.